src/test/java/org/openrewrite/java/dropwizard/annotation/security/PreAuthorizeTest.java

```diff
@@ -0,0 +1,138 @@
+/*
+ * Copyright 2025 the original author or authors.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.openrewrite.java.dropwizard.annotation.security;
+
+import org.junit.jupiter.api.Test;
+import org.openrewrite.DocumentExample;
+import org.openrewrite.java.JavaParser;
+import org.openrewrite.test.RecipeSpec;
+import org.openrewrite.test.RewriteTest;
+
+import static org.openrewrite.java.Assertions.java;
+
+class PreAuthorizeTest implements RewriteTest {
+
+    @Override
+    public void defaults(RecipeSpec spec) {
+        spec.recipes(new PermitAllToPreAuthorizeTransformer(), new RolesToPreAuthorize())
+          .parser(JavaParser.fromJavaVersion().classpath(JavaParser.runtimeClasspath()));
+    }
+
+    @DocumentExample
+    @Test
+    void transformsRolesAllowed() {
+        rewriteRun(
+          //language=java
+          java(
+            """
+              package com.example;
+
+              import javax.annotation.security.RolesAllowed;
```

Comment:
I think Spring Security supports these annotations as well; do we really have to convert these to PreAuthorize? Or can we leave these as is and configure Spring Security to support those annotations through `@EnableMethodSecurity(jsr250Enabled = true)`?